ckanext/aircan/logic/action.py

```
@@ -0,0 +1,178 @@
+            "api_key": tk.config.get("ckanext.aircan.ckan_api_key"),
+        },
+    }
+
+    for plugin in plugins.PluginImplementations(interfaces.IAircan):
+        plugin.update_payload(context, payload)
+
+    client = AirflowClient()
+    try:
+        dag_run = client.trigger_dag(conf=payload)
+    except requests.HTTPError as e:
+        log.error(tk._("Failed to trigger Airflow DAG '%s': %s"), client.dag_id, str(e))
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)